core/src/rpc/operator.rs

```diff
@@ -160,4 +157,44 @@ impl ClementineOperator for Operator {
 
         Ok(Response::new(Empty {}))
     }
+
+    #[tracing::instrument(skip(self), err(level = tracing::Level::ERROR), ret(level = tracing::Level::TRACE))]
+    async fn create_signed_tx(
```

Comment:
add an authentication guard, this should only be accessible from the local machine